Xamarin.PropertyEditing.Tests/CreateBindingViewModelTests.cs

```diff
@@ -192,7 +193,12 @@ public async Task ValueConverters ()
 
 			await vm.ValueConverters.Task;
 			Assert.That (vm.ValueConverters.Value, Contains.Item (visi));
-			Assert.That (vm.ValueConverters.Value.Count, Is.EqualTo (3)); // visi, No Converter, Request Converter
+
+			if (vm.IncludeAddValueConverter) {
```

Comment:
@ermau we were already checking that the `ValueConverters.Value.Count` was correct in this test. As `GetValueConvertersAsync ()` is called and therefore `AddValueConverter` is added or not, via the `CreateBindingViewModel` constructor. Do we still need separate tests?